doc/rst/language/spec/statements.rst

```diff
@@ -927,6 +929,208 @@ only applicable to modules, please see :ref:`Using_Modules`.
+--------------------
+
+The import statement provides either only qualified access to all of the public
+symbols of a module or only unqualified access to the specified public symbols
+of a module.
+
+The syntax of the import statement is given by:
+
+.. code-block:: syntax
+
+   import-statement:
+     privacy-specifier[OPT] `import' module-or-symbol-rename ;
+     privacy-specifier[OPT] `import' module-or-symbol-base unqualified-list ;
+
+   module-or-symbol-rename:
+     rename-base
+     identifier . module-or-symbol-rename
+
+   module-or-symbol-base:
+     identifier
+     identifier . module-or-symbol-base
+
+   unqualified-list:
+     . { rename-list }
+
+For example, the program
+
+   *Example (import1.chpl)*.
+
+   .. code-block:: chapel
+
+      module M1 {
+        proc foo() {
+          writeln("In M1's foo.");
+        }
+      }
+
+      module M2 {
+        import M1;
+        proc main() {
+          writeln("In M2's main.");
+          M1.foo();
+        }
+      }
+
+   prints out
+
+   .. code-block:: printoutput
+
+      In M2's main.
+      In M1's foo.
+
+This program is equivalent to:
+
+   *Example (import2.chpl)*.
+
+   .. code-block:: chapel
+
+      module M1 {
+        proc foo() {
+          writeln("In M1's foo.");
+        }
+      }
+
+      module M2 {
+        proc main() {
+          import M1.foo;
+
+          writeln("In M2's main.");
+          foo();
+        }
+      }
+
+   which also prints out
+
+   .. code-block:: printoutput
+
+      In M2's main.
+      In M1's foo.
+
+And both programs are also equivalent to:
+
+   *Example (import3.chpl)*.
+
+   .. code-block:: chapel
+
+      module M1 {
+        proc foo() {
+          writeln("In M1's foo.");
+        }
+      }
+
+      module M2 {
+        proc main() {
+          import M1.{foo};
+
+          writeln("In M2's main.");
+          foo();
+        }
+      }
+
+   which also prints out
+
+   .. code-block:: printoutput
+
+      In M2's main.
+      In M1's foo.
+
+The ``module-or-symbol-rename`` or ``module-or-symbol-base`` in an ``import``
+statement must begin with one of the following:
+
+ * a top-level module name
+ * a module name currently in scope due to another ``use`` or ``import``
+   statement
+ * any number of ``super`` components to indicate a number of parents of the
+   current module (e.g. ``super.super.SomeModule``)
+ * ``this`` to indicate the requested module is a submodule of the current
+   module
+
+A submodule may not be imported without either the full path to it, or a
+``super`` or ``this`` prefix at the beginning of the path.
+
+The names that are imported by an import statement are inserted in to a new
+scope that immediately encloses the scope within which the statement appears.
+This implies that the position of the import statement within a scope has no
+effect on its behavior.  If a scope includes multiple import statements, or a
+combination of import and use statements, then the imported names are inserted
```

Comment:
Again the second "import" here refers to both `use` and `import` and reads weird

Reply:
This should be resolved, let me know if you want me to tweak it more :)